recipe/meta.yaml

```diff
@@ -13,9 +13,9 @@ source:
   '{{ hash_type }}': '{{ hash_value }}'
 
 build:
-  number: 0
+  number: 1
   script: "{{ PYTHON }} -m pip install . --no-deps -vv"
-  skip: True # [py>36]
+  skip: True # [py<36]
```

Comment:
Not sure why it's showing this diff, the master branch has a correct condition now: https://github.com/nsls-ii-forge/immutables-feedstock/blob/b91e50415117e1e9146f93f2f98e5e0314ee621c/recipe/meta.yaml#L18

Any ideas?

Reply:
When I initially rendered with the wrong sign. I think some of my forced pushes made the history confusing.

Reply:
OK, thanks.